chain/consensus/filcns/compute_state.go

```diff
@@ -109,19 +108,6 @@ func (t *TipSetExecutor) ApplyBlocks(ctx context.Context, sm *stmgr.StateManager
 			LookbackState:  stmgr.LookbackStateGetterForTipset(sm, ts),
 		}
 
-		if os.Getenv("LOTUS_USE_FVM_EXPERIMENTAL") == "1" {
-			// This is needed so that the FVM does not have to duplicate the genesis vesting schedule, one
-			// of the components of the circ supply calc.
-			// This field is NOT needed by the LegacyVM, and also NOT needed by the FVM from v15 onwards.
-			filVested, err := sm.GetFilVested(ctx, e)
-			if err != nil {
-				return nil, err
-			}
-
-			vmopt.FilVested = filVested
-			return vm.NewFVM(ctx, vmopt)
-		}
-
 		return sm.VMConstructor()(ctx, vmopt)
```

Comment:
This should use the FVM automatically. As far as I can tell, we only needed the above check to set `FilVested`, which is no longer used.